src/CheckoutSdk/Payments/Contexts/PaymentContextsItems.cs

```diff
@@ -2,6 +2,7 @@ namespace Checkout.Payments.Contexts
 {
     public class PaymentContextsItems
     {
+        public PaymentContextItemType? Type { get; set; }
```

Comment:
Adding payment type is a must but this will not work with Klarna. There should be a discount type to be able to store discounts. Also if the DiscountAmount is not null, it will throw an error.

Reply:
As it says [here](https://www.checkout.com/docs/payments/add-payment-methods/klarna#Discount_example), to use any kind of voucher/discounts you should pass the following object to the items:
```json
"type": "discount", // This value should be present on PaymentContextItemType
"name": "The discount description",
"quantity": 1,
"reference": "VoucherCode", // This can be empty or null
"total_amount": 830, // The voucher discount
"unit_price": 830 // The voucher discount
```

Reply:
Thank you for your comments, I will fix the ints with nullable property in this class.